iOS READMEs - Features (#853)
Esri/arcgis-runtime-samples-ios: arcgis-ios-sdk-samples/Features/Feature layer rendering mode (scene)/README.md

In static rendering mode, features and graphics are rendered only when needed (for example, after an extent change) and offloads a significant portion of the graphical processing onto the CPU. As a result, less work is required by the GPU to draw the graphics, and the GPU can spend its resources on keeping the UI interactive. Use this mode for stationary graphics, complex geometries, and very large numbers of features or graphics. The number of features and graphics has little impact on frame render time, meaning it scales well, and pushes a constant GPU payload. However, rendering updates is CPU and system memory intensive, which can have an impact on device battery life.

## How to use the sample

> Use the 'Zoom In' button to trigger the same zoom animation on both static and dynamicly rendered scenes.

Reviewer: ```suggestion
Use the "Zoom In" button to trigger the same zoom animation on both static and dynamicly rendered scenes.
```
Double quote?

> Use the "Zoom In" button to trigger the same zoom animation on both static and dynamicly rendered scenes.

Reviewer: ```suggestion
Use the "Zoom In" button to trigger the same zoom animation on both static and dynamically rendered scenes.
```
typo.

## How it works

> 1. Create an `AGSSceneView` for each staic and dynamic mode and create an `AGSScene` for each.

Reviewer: ```suggestion
1. Create an `AGSSceneView` for each static and dynamic mode and create an `AGSScene` for each.
```
typo
